File: Battelle.EPA.WideAreaDecon.API/Enumeration/Parameter/ParameterType.cs
using System.Runtime.Serialization;
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;

namespace Battelle.EPA.WideAreaDecon.API.Enumeration.Parameter
{
    [JsonConverter(typeof(StringEnumConverter))]
    public enum ParameterType
    {
        [EnumMember(Value = "Constant")] Constant,

        [EnumMember(Value = "Uniform")] Uniform,

        [EnumMember(Value = "Uniform X Dependent")]
        UniformXDependent,

        [EnumMember(Value = "Beta PERT")] PERT,

        [EnumMember(Value = "Truncated Normal")]
        TruncatedNormal,

        [EnumMember(Value = "Bimodal Truncated Normal")]
        BimodalTruncatedNormal,

        [EnumMember(Value = "Log Uniform")] LogUniform,

        [EnumMember(Value = "Truncated Log Normal")]
        TruncatedLogNormal,

        [EnumMember(Value = "Log Normal")] LogNormal,

        [EnumMember(Value = "Weibull")] Weibull,

        [EnumMember(Value = "Enumerated Fraction")]
        EnumeratedFraction,

        [EnumMember(Value = "Enumerated Parameter")]
        EnumeratedParameter,

        [EnumMember(Value = "Null")] Null
    }
}

File: Battelle.EPA.WideAreaDecon.Client/src/enums/parameter/parameterType.ts
// eslint-disable-next-line no-shadow
enum ParameterType {
  constant = 'Constant',
  uniform = 'Uniform',
  uniformXDependent = 'Uniform X Dependent',
  pert = 'Beta PERT',
  scaledBeta = 'Scaled Beta',
  truncatedNormal = 'Truncated Normal',
  bimodalTruncatedNormal = 'Bimodal Truncated Normal',
  logUniform = 'Log Uniform',
  truncatedLogNormal = 'Truncated Log Normal',
  logNormal = 'Log Normal',
  weibull = 'Weibull',
  textValue = 'Text',
  enumeratedFraction = 'Enumerated Fraction',
  enumeratedParameter = 'Enumerated Parameter',
  contaminationDefinition = 'Contamination Definition',
  null = 'Null',
}

export default ParameterType;
